package.json

```diff
@@ -3,23 +3,28 @@
   "version": "0.1.0",
   "private": true,
   "devDependencies": {
+    "babel-jest": "^18.0.0",
     "axios": "^0.15.3",
     "babel-loader": "^6.2.10",
     "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
     "bootstrap": "^3.3.7",
+    "jest": "^18.1.0",
     "react": "^15.4.1",
     "react-bootstrap": "^0.30.7",
     "react-dom": "^15.4.1",
     "react-modal": "^1.6.5",
     "react-router": "^3.0.0",
     "react-scripts": "0.8.4",
+    "react-test-renderer": "^15.4.2",
     "react-select": "^1.0.0-rc.2",
     "serve": "^2.1.2",
     "webpack": "^1.14.0"
   },
   "dependencies": {
+    "enzyme": "^2.7.0",
     "react": "^15.4.1",
+    "react-addons-test-utils": "^15.4.2",
```

Comment:
Can we keep the test dependencies out of here?  
I've been looking at this" https://docs.npmjs.com/files/package.json#dependencies
I'm also going to add a story to clean up the dependencies to see if we can compile faster.